src/types/requests/sender.ts

```diff
@@ -0,0 +1,28 @@
+import { Sender } from '../models';
+import { RecursivePartial } from '../utils';
+
+/** @public */
+export interface GetAllSendersParameters {
+  /** Any text which is going to be used in conjunction with the searchtype if provided */
+  search?: string;
+
+  /** The first record that will be returned. Useful for pagination. */
+  from?: number;
+
+  /** The last record that will be returned. Useful for pagination. */
+  to?: number;
+
+  /** The direction according to which the data will be sorted. 'asc' for ascending and 'desc' for descending. */
+  dir?: 'asc' | 'desc';
```

Comment:
```suggestion
  dir?: 'ASC' | 'DESC';
```

nit - maybe to be consistent with the other string literal union types?

Reply:
Same here - OneSpan only accepts these values in lowercase ([docs](https://community.onespan.com/products/onespan-sign/sandbox#/Senders/api.account.senders.get))

Reply:
Cool, let's keep it as is then~